### PR TITLE
adds a brainrot line to the brain damage lines

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
"%STICKING OUT YOUR GYATT FOR THE RIZZLER!!" is now another brain damage line
## Why It's Good For The Game
I think brain-damaged people should be really annoying.
## Changelog
:cl: grungussuss
add: "brain damaged people now have brainrot from tik tok
/:cl:

### CODE REVIEW

**Comment by zoomachina** [comment]
nononono

**Comment by Loafin34** [comment]
can you add "Scream if you like Space Station 13"

**Comment by projectkepler-ru** [comment]
"host of (x) server caught trying to remove content people actually enjoy using, get absolutely dunked on by maintainers" could be a line idk

**Comment by Sadboysuss** [comment]
> "host of (x) server caught trying to remove content people actually enjoy using, get absolutely dunked on by maintainers" could be a line idk

irrelevant to this pr

File: strings/traumas.json [hunk 1]
```diff
@@ -68,7 +68,8 @@
         "who the HELL do u thenk u r?!!!!",
         "without oxigen blob don't evoluate?",
         "youed call her a toeugh bithc",
-        "I AM ALPHA MAN!!"
+        "I AM ALPHA MAN!!",
+		"%STICKING OUT YOUR GYATT FOR THE RIZZLER!!"
```
Comment (ZephyrTFA) [inline strings/traumas.json:72]: fix your spacing

**Comment by optimumtact** [comment]
mmm no thanks.

**Pull request closed without merging**